Apps/Examples/Examples/All Examples/AnimateLayerExample.swift

```diff
@@ -108,12 +108,12 @@ public class AnimateLayerExample: UIViewController, ExampleProtocol {
 
             // Identify the new coordinate to animate to, and calculate
             // the bearing between the new coordinate and the following coordinate.
-            var geoJSON = Feature.init(geometry: Geometry.point(Point(coordinate)))
-            geoJSON.properties = ["bearing": coordinate.direction(to: nextCoordinate)]
+            var geoJSON = Feature(geometry: .point(Point(coordinate)))
```

Comment:
Something to consider for Turf in the future is introducing a protocol like GEOSwift's GeometryConvertible so that we can avoid the `.point()` wrapper in situations like this. It would make the code look like…

```suggestion
            var geoJSON = Feature(geometry: Point(coordinate))
```

Reply:
I suppose we could do the same for GeoJSON as well to avoid having to write out those enum wrappers as often.

Reply:
Good idea! I opened mapbox/turf-swift#158 to track this change. It’d be out of scope for this PR because it would require upgrading to a subsequent version of Turf. In the meantime, I’m also open to writing this as designated initializer shorthand for less redundancy:

```swift
var geoJSON = Feature(geometry: .point(.init(coordinate)))
```

but that might be my Objective-C++ background showing through, maybe too obtuse for an example.

Reply:
😄 I have mixed feelings about type inference with `init`. We use it somewhat inconsistently at the moment. Let's leave this as-is for now, and I'll make a note about coming up with some guidelines around this for the future.